automation/post_discussion.py

```diff
@@ -1,23 +1,22 @@
 import huggingface_hub as hf_hub
 import markdown
+import re
+from time import sleep
 
-
-def construct_post_content(
-    report, dataset_id, dataset_config, dataset_split, scan_report
-):
-    vulnerability_count = len(scan_report.scan_result.issues)
-
-    # Construct the content of the post
+def construct_opening(dataset_id, dataset_config, dataset_split, vulnerability_count):
     opening = """
     \nHey Team!🤗✨ <br />We’re thrilled to share some amazing evaluation results that’ll make your day!🎉📊<br />
     """
     opening += f"""
-\nWe have identified {vulnerability_count} potential vulnerabilities in your model based on an automated scan.
+    \nWe have identified {vulnerability_count} potential vulnerabilities in your model based on an automated scan.
```

Comment:
The leading spaces are somehow also considered in the format string, which will give a quote block in markdown.

Reply:
Sorry, I don't understand. This will not make it a quote block as shown in the demo or this link: https://huggingface.co/spaces/ZeroCommand/test-giskard-bot/discussions/42

There would be a tab followed by the line breaker, but should be fine because it would not be visible.